geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/AlterRegionCommandDUnitTest.java

```diff
@@ -64,10 +69,20 @@ public static void beforeClass() throws Exception {
     gfsh.connectAndVerify(locator);
   }
 
-  @Test
-  public void testAlterRegionResetCacheListeners() throws IOException {
+  @Before
+  public void before() throws Exception {
+    // make sure all tests started with no region defined
```

Comment:
Since we expect it at past of the `@After`, a comment here might be good stating that each individual test requires some kind of `create region` command.

Or that might be obvious from looking at the class.  I don't know.  It's been a long day.

Reply:
this is to make sure each test starts with a clean slate. the test alone might not need to create any region at all.